src/Game.java

```diff
@@ -1,88 +1,89 @@
 import java.util.Scanner;
 
-
 public class Game {
 
-    private char lastGuessedLetter;
-    private int guessesLeft;
-    private boolean lastGuessCorrect;
-    private boolean hasWon;
-
-    Game(){
-        guessesLeft = 10;
-    }
-
-    public void playturn(String randomMovieTitle){
-
-        this.setUserGuess();
-        this.setLetterExists(randomMovieTitle, this.lastGuessedLetter);
-        this.setGuessesLeft();
-
-    }
-
-
-    private void setUserGuess(){
-
-        Scanner userInput = new Scanner(System.in);
-
-        char guessedLetter = userInput.nextLine().toLowerCase().charAt(0);
-
-        this.lastGuessedLetter = guessedLetter;
-
-    }
-
-    public char getLastGuessedLetter(){
-        return this.lastGuessedLetter;
+  private char lastGuessedLetter;
+  private int guessesLeft;
+  private boolean lastGuessCorrect;
+  private boolean hasWon;
+  private boolean gameInPlay;
+  private Movie movie;
+
+  Game(Scanner movieList) {
+    guessesLeft = 10;
+    gameInPlay = true;
+    hasWon = false;
+    movie = new Movie(movieList);
+  }
+
+  public void playturn(Scanner userInput) {
```

Comment:
In the issue I raised, I meant that both `Game` and `Movie` would have no relationship with reading directly from either disk or the standard input.
What this would look like is that both classes would have no relationship with `Scanner` at all, instead `Movie` would take a list of movies to work with and `playturn` in `Game` would take a guess; these classes don't have to know where the data has come from.